frontend/src/metabase/visualizations/components/ScalarValue/ScalarValue.styled.tsx

```diff
@@ -54,14 +59,18 @@ export const ScalarTitleContent = styled.h3`
 export const ScalarDescriptionContainer = styled.div`
   cursor: pointer;
   width: 1.5rem;
-  margin-top: 0.25rem;
+  margin-top: 0.2rem;
   padding-left: 0.5rem;
-
-  &:hover {
-    color: ${color("brand")};
-  }
 `;
 
 export const ScalarDescriptionPlaceholder = styled.div`
   width: 1.5rem;
+  margin-top: 0.2rem;
+  padding-right: 0.5rem;
```

Comment:
There isn't an RFC about this yet, but this is a part of introducing Metabase components or what we've been doing in the project UI library. [Here's some of the conversation](https://metaboat.slack.com/archives/C057WD5L0JG/p1686227773463059?thread_ts=1686219268.256349&cid=C057WD5L0JG), and [a PR that starts using `Box`](https://github.com/metabase/metabase/pull/31426) component in our app (the description is outdated though, nothing there is about `createStyles`)

I think the thing we're trying to avoid is to stop positioning elements inside the element itself if that makes sense. Generally, I view a component as a standalone UI without either margin or padding. And the parent components that want to use any specific UI need to know where to position those UI themselves that's why I think leaving spacing e.g. `margin` and `padding` in the parent component would make more sense.

This will enable us to use reusable components with less friction as well, previously we need to create a new `*.styled.*` file if it doesn't exist. Extend the component you want to use with `styled(YourComponent)` then add one or 2 lines for the spacing. That won't be necessary with `Box` and `Flex` and all other layout components Mantine provide anymore.

Reply:
Thank you for a detailed explanation! :100: 

If you don't mind I will skip this until we establish a convention.

Reply:
Sounds fine.